src/pretalx/orga/views/schedule.py

```diff
@@ -141,6 +142,7 @@ def form_invalid(self, form):
     def form_valid(self, form):
         self.request.event.release_schedule(
             form.cleaned_data["version"],
+            form.cleaned_data["comment"],
```

Comment:
Just a reminder, as per my other comments I'd prefer keyword arguments for this method, so please adjust the call here, too.